meshtastic/tests/test_api_baseline_comparison.py

```
@@ -0,0 +1,718 @@
+        "meshtastic.ota",
+        "meshtastic.remote_hardware",
+        "meshtastic.slog",
+        "meshtastic.tcp_interface",
+        "meshtastic.test",
+        "meshtastic.tunnel",
+    ]
+    for module_name in modules_to_preimport:
+        try:
+            importlib.import_module(module_name)
+        except ImportError:
+            pass
```

Ruff (0.15.7)

[warning] Use `contextlib.suppress(ImportError)` instead of `try`-`except`-`pass`

Replace `try`-`except`-`pass` with `with contextlib.suppress(ImportError): ...`

(SIM105)